build-tools/xaprepare/xaprepare/ToolRunners/GitRunner.cs

```diff
@@ -187,10 +207,20 @@ public async Task<bool> IsRepoUrlHttps (string workingDirectory)
 			return containsHttps;
 		}
 
-		ProcessRunner CreateGitRunner (string? workingDirectory, List<string>? arguments = null)
+		ProcessRunner CreateGitRunner (string? workingDirectory, List<string>? arguments = null, bool useCustomStderrWrapper = false)
 		{
 			var runner = CreateProcessRunner ();
 			runner.WorkingDirectory = workingDirectory;
+			if (useCustomStderrWrapper) {
+				runner.StandardErrorEchoWrapper = new GitProgressStderrWrapper {
+					LoggingLevel = runner.EchoStandardErrorLevel,
+					CustomSeverityName = ProcessRunner.StderrSeverityName,
+				};
+			}
+
+			// We need English for line matching in the stderr handler
+			runner.Environment ["LANG"] = "C";
```

Comment:
I don't think that this line is required anymore, as we're not comparing the messages to anything, just capturing them.